src/repositories/hive/hive.repository.ts

```diff
@@ -3,42 +3,69 @@ import hiveJsPackage from '@hiveio/hive-js';
 import { AuthorPerm, OperationsArray } from "./types";
 import { Client, ExtendedAccount, Operation, PrivateKey, PublicKey, Signature } from '@hiveio/dhive';
 import crypto from 'crypto'
+import 'dotenv/config'
```

Comment:
Ah, clever; I'd been using Nest's ConfigService, but its awesome if this works! We should aim to have a unified approach across the codebase though